src/Microsoft.Azure.WebJobs.Host/Scale/IScalerFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    public interface IScalerFactory
```

Comment:
Not sure this interface needs to be in the SDK. I don't see it used from either the SDK or the Host PR you linked to, so likely it's a SC specific thing and can just live there.

Reply:
All the extensions will implement the interface to use it from ScaleController. WebJobs SDK is common part for all the extensions.